### ISSUE TITLE
cytokine, chemokine, and chemokine receptor terms

### ISSUE THREAD

**Issue by slaulederkind** [mgijax/mammalian-phenotype-ontology#4209]
Preferred term label (e.g., asplenia)

1. abnormal Xcl1 level
2. increased Xcl1 level
3. decreased Xcl1 level
4. abnormal Ccl4 level
5. increased Ccl4 level
6. decreased Ccl4 level
7. abnormal chemokine receptor level
8. abnormal Cxcr3 level
9. increased Cxcr3 level
10. decreased Cxcr3 level
11. abnormal Ccr5 level
12. increased Ccr5 level
13. decreased Ccr5 level
14. increased interferon-gamma level 
15. decreased interferon-gamma level
16. increased Il2 level
17. decreased Il2 level
18. increased Il4 level
19. decreased Il4 level
20. increased Il5 level
21. decreased Il5 level
22. increased Il13 level
23. decreased Il13 level

2. Synonyms (e.g., absent spleen)

1. abnormal X-C motif chemokine ligand 1 level
2. increased X-C motif chemokine ligand 1 level
3. decreased X-C motif chemokine ligand 1 level
4. abnormal C-C motif chemokine ligand 4 level; abnormal Mip1b level; abnormal macrophage inflammatory protein-1 beta level
5. increased C-C motif chemokine ligand 4 level; increased  Mip1b level; increased macrophage inflammatory protein-1 beta level
6. decreased C-C motif chemokine ligand 4 level; decreased  Mip1b level; decreased macrophage inflammatory protein-1 beta level
8. abnormal C-X-C motif chemokine receptor 3 level; abnormal G protein-coupled receptor 9 level; abnormal Gpr9 level
9. increased C-X-C motif chemokine receptor 3 level; increased G protein-coupled receptor 9 level; increased Gpr9 level
10. decreased C-X-C motif chemokine receptor 3 level; decreased G protein-coupled receptor 9 level; decreased Gpr9 level
11. abnormal C-C motif chemokine receptor 5 level; abnormal Mip1a receptor level; abnormal macrophage inflammatory protein-1 alpha receptor level
12. increased C-C motif chemokine receptor 5 level; increased Mip1a receptor level; increased macrophage inflammatory protein-1 alpha receptor level
13. decreased C-C motif chemokine receptor 5 level; decreased Mip1a receptor level; decreased macrophage inflammatory protein-1 alpha receptor level
14. increased IFN-gamma level; increased Ifng level
15. decreased IFN-gamma level; decreased Ifng level
16. increased interleukin 2 level; increased T-cell growth factor level, increased Tcgf level
17. decreased interleukin 2 level; decreased T-cell growth factor level, decreased Tcgf level
18. increased interleukin 4 level; increased B-cell stimulatory factor 1 level; increased Bsf1 level
19. decreased interleukin 4 level; decreased B-cell stimulatory factor 1 level; decreased Bsf1 level
20. increased interleukin 5 level; increased B-cell growth factor II level; increased Bcgf-II level
21. decreased interleukin 5 level; decreased B-cell growth factor II level; decreased Bcgf-II level
22. increased interleukin 13 level; increased T-cell activation protein P600 level
23. decreased interleukin 13 level; decreased T-cell activation protein P600 level

3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

1. deviation from the normal levels of X-C motif chemokine ligand 1, involved in leukocyte migration and chemotaxis.
2. elevated levels of X-C motif chemokine ligand 1, involved in leukocyte migration and chemotaxis.
3. reduced levels of X-C motif chemokine ligand 1, involved in leukocyte migration and chemotaxis.
4. deviation from the normal levels of C-C motif chemokine ligand 4, a chemokine acting through Ccr1 or Ccr5 receptors.
5. elevated levels of C-C motif chemokine ligand 4, a chemokine acting through Ccr1 or Ccr5 receptors.
6. reduced levels of C-C motif chemokine ligand 4, a chemokine acting through Ccr1 or Ccr5 receptors.
7. deviation from the normal levels of chemokine receptors, proteins which bind to and transduce signals of chemokines.
8. deviation from the normal levels of C-X-C motif chemokine receptor 3, a receptor involved in T cell chemotaxis.
9. elevated levels of C-X-C motif chemokine receptor 3, a receptor involved in T cell chemotaxis.
10. reduced levels of C-X-C motif chemokine receptor 3, a receptor involved in T cell chemotaxis.
11. deviation from the normal levels of C-C motif chemokine receptor 5, a protein involved in negative regulation of axon extension, positive regulation of cytokine production, and response to bacterium.
12. elevated levels of C-C motif chemokine receptor 5, a protein involved in negative regulation of axon extension, positive regulation of cytokine production, and response to bacterium.
13. reduced levels of C-C motif chemokine receptor 5, a protein involved in negative regulation of axon extension, positive regulation of cytokine production, and response to bacterium.
14. elevated levels of interferon gamma, a cytokine involved in microglial cell activation, negative regulation of cell population proliferation, and negative regulation of epithelial cell differentiation.
15. reduced levels of interferon gamma, a cytokine involved in microglial cell activation, negative regulation of cell population proliferation, and negative regulation of epithelial cell differentiation.
16. elevated levels of interleukin 2, a cytokine produced by by antigen- or mitogen-stimulated T-cells which induces DNA synthesis in naive lymphocytes.
17. reduced levels of interleukin 2, a cytokine produced by by antigen- or mitogen-stimulated T-cells which induces DNA synthesis in naive lymphocytes.
18. elevated levels of interleukin 4, a cytokine involved in microglial cell activation, positive regulation of leukocyte migration, and regulation of defense response.
19. reduced levels of interleukin 4, a cytokine involved in microglial cell activation, positive regulation of leukocyte migration, and regulation of defense response.
20. elevated levels of interleukin 5, a cytokine involved in positive regulation of B cell proliferation.
21. reduced levels of interleukin 5, a cytokine involved in positive regulation of B cell proliferation.
22. elevated levels of interleukin 13, a cytokine involved in positive regulation of cell population proliferation and microglial cell activation.
23. reduced levels of interleukin 13, a cytokine involved in positive regulation of cell population proliferation and microglial cell activation.

[PMID 29564567](https://urldefense.com/v3/__https://rgd-jira.atlassian.net/browse/OTR-122__;!!H8mHWRdzp34!7Wmfe3KeqcodK1fc-X13zXIv-fRuHCZQ9EiXhUlcBsKCB8JnXYIgbIBGHumNdy3vjISvoh2U7cZ5Pbe3RgEoHg$)

4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

1. abnormal chemokine level (MP:0008721)
2. abnormal Xcl1 level (new term)
3. abnormal Xcl1 level (new term)
4. abnormal chemokine level (MP:0008721)
5. abnormal Ccl4 level (new term)
6. abnormal Ccl4 level (new term)
7. abnormal protein level (MP:0008469)
8. abnormal chemokine receptor level (new term)
9. abnormal Cxcr3 level (new term)
10. abnormal Cxcr3 level (new term)
11. abnormal chemokine receptor level (new term)
12. abnormal Ccr5 level (new term)
13. abnormal Ccr5 level (new term)
14. increased cytokine level (MP:0014433)
15. decreased cytokine level (MP:0014434)
16. increased cytokine level (MP:0014433)
17. decreased cytokine level (MP:0014434)
18. increased cytokine level (MP:0014433)
19. decreased cytokine level (MP:0014434)
20. increased cytokine level (MP:0014433)
21. decreased cytokine level (MP:0014434)
22. increased cytokine level (MP:0014433)
23. decreased cytokine level (MP:0014434)

6.  If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

ORCID:0000-0001-5356-4174

**Comment by sbello** [comment]
made it to decreased Il2 level, will finish tomorrow
